mmdet/datasets/pipelines/loading.py

```diff
@@ -550,18 +550,24 @@ class FilterAnnotations:
             boxes.
     """
 
-    def __init__(self, min_gt_bbox_wh):
+    def __init__(self, min_gt_bbox_wh, always_keep=False):
```

Comment:
What does `always_keep` mean?

Reply:
I didn't expect a better name. Its meaning is that when the filtering rule becomes an empty box, the filtering rule is skipped at this time, and the result before the filter is always retained.

Reply:
how about keep_empty?

Reply:
OK